### DISCUSSION TITLE
Can't install mlvus_cli with pymilvus 2.4.4

### DISCUSSION THREAD

**Discussion by nikhilreddy-pottanig** [milvus-io/milvus#36646] [Q&A and General discussion]
ERROR: Cannot install milvus-cli==1.0.0 and pymilvus==2.4.4 because these package versions have conflicting dependencies.

**Comment by xiaofan-luan** [comment]
can you list what the error is? 

from the docuemnt this is compatible
https://milvus.io/docs/cli_overview.md
